teuthology/orchestra/remote.py

```diff
@@ -401,6 +401,15 @@ def get_file(self, path, sudo=False, dest_dir='/tmp'):
             self.remove(path)
         return local_path
 
+    def write_file(self, path, data, sudo=False):
+        """
+        Write a file on a remote machine.
+        """
+        args = 'sudo tee' if sudo else 'tee'
+        tempfilename = tempfile.mkstemp()[1]
+        self.run('%s %s' % (args, tempfilename), stdin=data)
```

Comment:
You do not need `tempfile` here, since it `self.run` is already executed on a remote host and you can give it a destination file path.

Reply:
Right. I've made the change, write_file does `sudo tee path`/`tee path` now.